packages/grafana-e2e/src/pages/login/index.ts

```diff
@@ -0,0 +1,14 @@
+import { pageFactory } from '../../support';
+
+const selectors = {
+  username: 'Username input field',
+  password: 'Password input field',
+  submit: 'Login button',
+};
```

Comment:
These selectors will be used in Grafana components so that everything is the same. Working on that now, have some build errors to handle to get that to work.